docs/build/references/command-line-interface.md

```diff
@@ -383,7 +383,10 @@ The default C-Chain config is:
   "remote-tx-gossip-only-enabled": false,
   "tx-regossip-frequency": 60000000000,
   "tx-regossip-max-size": 15,
-  "log-level": "debug"
+  "log-level": "debug",
+  "offline-pruning-enabled": false,
+  "offline-pruning-bloom-filter-size": 512, // MB
+  "offline-pruning-data-directory": "",
```

Comment:
should we set the default dir to `/tmp`?

Reply:
Currently it just defaults to the empty string so I think we should document it as such for now. Open to a PR to update the default directory in coreth though as long as we make sure it works on all our supported OSs.